packages/beacon-node/src/chain/lightClient/index.ts

```diff
@@ -291,6 +293,31 @@ export class LightClientServer {
     return update;
   }
 
+  /**
+   * API ROUTE to get the sync committee hash from the best available update for `period`.
+   */
+  async getCommitteeHash(period: number): Promise<Uint8Array> {
+    const {attestedHeader} = await this.getUpdate(period);
+    const blockRoot = ssz.phase0.BeaconBlockHeader.hashTreeRoot(attestedHeader);
+
+    const syncCommitteeWitness = await this.db.syncCommitteeWitness.get(blockRoot);
+    if (!syncCommitteeWitness) {
+      throw new LightClientServerError(
+        {code: LightClientServerErrorCode.RESOURCE_UNAVAILABLE},
+        `syncCommitteeWitness not available ${toHexString(blockRoot)} period ${period}`
+      );
+    }
+
+    const currentSyncCommittee = await this.db.syncCommittee.get(syncCommitteeWitness.currentSyncCommitteeRoot);
+    if (!currentSyncCommittee) {
+      throw new LightClientServerError(
+        {code: LightClientServerErrorCode.RESOURCE_UNAVAILABLE},
+        `currentSyncCommittee not available ${toHexString(blockRoot)} for period ${period}`
+      );
+    }
+    return digest(concat(currentSyncCommittee.pubkeys));
```

Comment:
Do we agree on doing hashTreeRoot here or not?

Reply:
I can switch to use root unless @shresthagrawal wants to use hash for now

Reply:
Yes please go ahead with the root!